examples/composite_template/ex_menu_button/ex_menu_button.ui

```diff
@@ -6,6 +6,7 @@
     </property>
     <child>
       <object class="GtkToggleButton" id="toggle">
+        <signal name="toggled" handler="toggle_toggled" swapped="true"/>
```

Comment:
Do you always have to specify `swapped="true"` with the current implementation?

Reply:
No, see the macro code

Reply:
Indeed